content/docs/architecture/resources/environment.md

```diff
@@ -1,66 +1,87 @@
 +++
 title = "Environment"
+description = "Defines specific implementation details on how HobbyFarm connects to a provider for scheduling virtual machines."
 +++
 
-An `Environment` defines specific implementation details on how HobbyFarm connects to a provider for scheduling virtual machines. An environment resource contains configuration information such as where provider credentials are stored, what image to use when creating a VM, or how much capacity a provider has for a specific type of VM. 
 
-Here's an example environment manifest:
+An `Environment` resource contains configuration information such as where provider credentials are stored, what image to use when creating a VM, or how much capacity a provider has for a specific type of VM.
+
+## Kubernetes Commands
+The following commands are useful for managing Environment resources in Kubernetes.
+
+```bash
+## Get a list of all Environments
+kubectl get environments -n hobbyfarm-system
+
+## Create an Environment from a YAML manifest
+kubectl apply -f {environmentManifest} -n hobbyfarm-system
+
+## Edit an Environment
+kubectl edit environment {environmentName} -n hobbyfarm-system
+
+## Backup an Environment to a YAML manifest
+kubectl get environment {environmentName} -n hobbyfarm-system -o yaml > {environmentManifest}
+
+## Delete an Environment
+kubectl delete environment {environmentName} -n hobbyfarm-system
+```
+
+
+## Environment Manifest Example
+The following shows an example of an Environment manifest in Kubernetes.
 
 ```yaml
 apiVersion: hobbyfarm.io/v1
 kind: Environment
 metadata:
-    name: sample-environment
-    namespace: hobbyfarm-system
+  annotations:
+    hobbyfarm.io/provisioner: external
```

Comment:
Required only if using an external provisioner. This will break terraform functionality in Gargantua. It's slated for removal so it's not a _huge_ deal but could result in misconfig if someone really wants to use tf

Reply:
@ebauman I'll add a note about the TF breakage. Given the depreciation of TF, I didn't see it as being too important to note, but I see your point.